ape_etherscan/client.py

```diff
@@ -67,6 +67,8 @@ def get_etherscan_uri(ecosystem_name: str, network_name: str):
             if network_name != "mainnet"
             else "https://bscscan.com"
         )
+    elif ecosystem_name == "gnosis" and network_name=="mainnet":
+        return "https://gnosisscan.io"
 
     raise UnsupportedEcosystemError(ecosystem_name)
```

Comment:
Hmm, now I'm thinking this error should be changed a bit to show the combo isn't supported